packages/cli-repl/src/mongosh-repl.ts

```diff
@@ -173,12 +173,15 @@ class MongoshNodeRepl implements EvaluationListener {
     instanceState.setEvaluationListener(this);
     await instanceState.fetchConnectionInfo();
 
-    let mongodVersion = instanceState.connectionInfo.buildInfo?.version;
-    const apiVersion = serviceProvider.getRawClient()?.serverApi?.version;
-    if (apiVersion) {
-      mongodVersion =
-        (mongodVersion ? mongodVersion + ' ' : '') +
-        `(API Version ${apiVersion})`;
+    const { buildInfo, extraInfo } = instanceState.connectionInfo;
+    let mongodVersion = extraInfo?.is_stream ? undefined : buildInfo?.version;
```

Comment:
Are streams endpoint fully unversioned? Why does the `buildInfo` command report a version from the endpoint then? Having an endpoint version available is usually one of the most important things for debugging, so I’d like to make sure we’re omitting it in the right way. It probably makes sense to at least do something like

```suggestion
    let mongodVersion = extraInfo?.is_stream ? 'Atlas Streams Processing' : buildInfo?.version;
```

Reply:
Streams currently doesn't have version. Streams reuse the ADF FE and the `version` returned in `buildInfo` is the ADF version.

Reply:
Okay, then I’d still probably leave everything here more or less as it is and use the suggestion I made above?

When users share screenshots or terminal logs with us or TS, it’s going to be much easier to identify that they’re connecting to a streams processing instance if we just explicitly say `Using MongoDB: Atlas Streams Processing` instead of omitting the line entirely.

Reply:
Okay, have updated as what you suggested.